src/include/sig.h

```diff
@@ -182,7 +182,7 @@ void signal_return_to_dpmi(void);
 void signal_set_altstack(int on);
 void signative_init(void);
 void signative_pre_init(void);
-int signative_enter_skip_unblock(sigcontext_t *scp, unsigned long uc_flags);
```

Comment:
Seems like removing the func that you
added in a prev patch?

Reply:
yes, that's what I meant with unclean history :)